mainnet1/genesis.json

```diff
@@ -363,7 +363,15 @@
       "spam.protection.max.proposals": "3",
       "spam.protection.proposal.min.tokens": "200000000000000000000000",
       "spam.protection.max.delegations": "360",
-      "spam.protection.delegation.min.tokens": "1"
+      "spam.protection.delegation.min.tokens": "1",
```

Comment:
I am assuming that 
"spam.protection.delegation.min.tokens": "1",
means as usual that for this to be 1 VEGA we'd want 1 followed by 18 zeros, correct?

I'd suggest we set it to 0.1 VEGA i.e. 1 followed by 17 zeros.

Reply:
addressed